matRad_calcLateralParticleCutOff.m

```diff
@@ -79,14 +79,19 @@
          
         if strcmp(machine.meta.dataType,'singleGauss')
                     
-                    Sigma1 = machine.data(energyIx).sigma(Idx(j));
+                    Sigma1 = sqrt(machine.data(energyIx).sigma(Idx(j))^2 + machine.data(energyIx).initFocus.sigma(1)^2);
```

Comment:
Why dont you do the interpolation here as well?

Maybe in the future we want to have a change of the initial beam width also incorporated for the single gaussians. If we include that right now also in the code we have that flexibility automatically at a later point as well.